config/codeceptjs_helpers.js

```diff
@@ -65,60 +65,56 @@ class customHelpers extends Helper {
   }
 
-  // Custom dontSeeDisabledAttributet function.
+  // Custom dontSeeDisabledAttribute function.
   // Evaluates true if xpath is not disabled
   async dontSeeDisabledAttribute (xpath) {
     const helper = this.helpers['Puppeteer'] || this.helpers['WebDriver']
-    let res = await helper.grabAttributeFrom(xpath, 'disabled')
-    return assert.ok(!(res !== null && typeof res !== 'undefined' && res.toString().toLowerCase() === 'true'), "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' NOT to be disabled")
+    let res = await helper.grabAttributeFromAll(xpath, 'disabled')
+    return assert.ok(!(Array.isArray(res) && res.length > 0 && res[0].toString().toLowerCase() === 'true'), "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' NOT to be disabled")
   }
 
   // Custom seeReadOnlyAttribute function.
   // Evaluates true if xpath is disabled
   async seeReadOnlyAttribute (xpath) {
     const helper = this.helpers['Puppeteer'] || this.helpers['WebDriver']
-    let res = await helper.grabAttributeFrom(xpath, 'readonly')
-    return assert.ok(res !== null && typeof res !== 'undefined' && res.toString().toLowerCase() === 'true', "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' to be readonly")
+    let res = await helper.grabAttributeFromAll(xpath, 'readonly')
+    return assert.ok(Array.isArray(res) && res.length > 0 && res[0].toString().toLowerCase() === 'true', "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' to be readonly")
   }
 
-  // Custom dontSeeReadOnlyAttributet function.
+  // Custom dontSeeReadOnlyAttribute function.
   // Evaluates true if xpath is not disabled
   async dontSeeReadOnlyAttribute (xpath) {
     const helper = this.helpers['Puppeteer'] || this.helpers['WebDriver']
-    let res = await helper.grabAttributeFrom(xpath, 'readonly')
-    return assert.ok(!(res !== null && typeof res !== 'undefined' && res.toString().toLowerCase() === 'true'), "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' NOT to be readonly")
+    let res = await helper.grabAttributeFromAll(xpath, 'readonly')
+    return assert.ok(!(Array.isArray(res) && res.length > 0 && res[0].toString().toLowerCase() === 'true'), "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' NOT to be readonly")
   }
 
   // Custom grabBooleanAttributeFrom function.
   // returns boolean value
   async grabBooleanAttributeFrom (xpath, attrib) {
     const helper = this.helpers['Puppeteer'] || this.helpers['WebDriver']
-    try {
-      let res = await helper.grabAttributeFrom(xpath, attrib)
-      return res !== null && typeof res !== 'undefined' && res.toString().toLowerCase() === 'true'
-    } catch (err) {
-      console.log('CodeceptJs Custom Helper "grabBooleanAttributeFrom" Error:', err)
-    }
+    let res = await helper.grabAttributeFromAll(xpath, attrib)
+    return Array.isArray(res) && res.length > 0 && res[0].toString().toLowerCase() === 'true'
   }
```

Comment:
This function isn't used anywhere in the current test suite. For consistency, this should be:
```
return assert.ok(Array.isArray(res) && res.length > 0 && res[0].toString().toLowerCase() === 'true', "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' with attribute " + attrib)
  
```